sigopt/local_run_context.py

```diff
@@ -0,0 +1,41 @@
+from .run_context import BaseRunContext
+import copy
+
+class LocalRunContext(BaseRunContext):
+  def __init__(self, **kwargs):
+    self.run = copy.deepcopy(kwargs) if kwargs else {}
```

Comment:
could you create a unit test for this? 
maybe just add a fixture
in `runs/test_context.py`

Reply:
Done.